lib/content_block_tools/content_block_reference.rb

```diff
@@ -40,7 +40,6 @@ class << self
       #
       # @return [Array<ContentBlockReference>] An array of content block references
       def find_all_in_document(document)
-        ContentBlockTools.logger.info("Finding embedded content")
         document.scan(ContentBlockReference::EMBED_REGEX).map do |match|
           ContentBlockTools.logger.info("Found Content Block Reference: #{match}")
```

Comment:
I'm keeping this one in for now because I think it will be useful to see when a Block is found, without it the whole process is basically invisible in the logs